Sources/LSPLogging/SplitLogMessage.swift

```diff
@@ -23,6 +23,12 @@ public func splitLongMultilineMessage(message: String, maxChunkSize: Int = 800)
     if let lastChunk = chunks.last, lastChunk.utf8.count + line.utf8.count < maxChunkSize {
       chunks[chunks.count - 1] += "\n" + line
     } else {
+      if !chunks.isEmpty {
+        // Append an end marker the the last chunks so that os_log doesn't truncate trailing whitespace,
```

Comment:
```suggestion
        // Append an end marker to the last chunk so that os_log doesn't truncate trailing whitespace,
```
... feel free to include that in another PR though.